lib/netproto/netproto_connect.c

```diff
@@ -129,6 +138,14 @@ getserveraddr(void)
 
 	/* If we have a new lookup, update the cache. */
 	if (tmp_addr != NULL) {
+
```

Comment:
No blank line here.  Yes, I know I said blank lines before comments.  But it's actually blank lines separating code from comments, and this is the start of a block so there's no preceding code at this indent level...